docs/Firewall-Automation.md

```diff
@@ -41,86 +136,75 @@ _The Azure CLI Automation extension is in an experimental stage.  Currently it d
+|  4. | Automation Runbook with Powershell        |                                      [link](https://learn.microsoft.com/en-us/azure/automation/automation-runbook-types#powershell-runbooks)                                     | Create a Runbook of type Powershell.                                                                                                                                        |
+|  5. | Powershell Content in Runbook             |                               [link](https://learn.microsoft.com/en-us/powershell/module/az.automation/import-azautomationrunbook?view=azps-8.3.0)                               | Upload [pre-defined Powershell content](../scripts/Firewall-Automation/Firewall-Automation-Runbook.ps1) into the Runbook body.                                                                            |
+|  6. | Automation Schedule(s) *using Powershell* |                               [link](https://learn.microsoft.com/en-us/powershell/module/az.automation/import-azautomationrunbook?view=azps-8.3.0)                               | Create the schedules that will execute the Firewall automation.  These had to be created using Powershell instead of the Azure CLI.  No equivalent behavior has been found. |
 
-### Resources Created When Complete
+## Resources Created When Complete
 
-1. Azure user-assigned Managed Identity
+1. Azure User-Assigned Managed Identity
 2. Azure Automation Account
 3. Azure PowerShell Runbook
 
     ![List of resources that will be created when complete.](./assets/Firewall-Automation/listOfResourcesInResourceGroup.png)
 
-## Installation Method - Automated Scripts
+## Runbook and Schedule Creation using Automated Scripts
 
-BEFORE continuing please make sure all requirements have been met in the section labeled [prerequisites]("#-prerequisites").
+The steps to set up the runbook and schedule are listed in this section. BEFORE continuing please make sure all requirements have been met in the section labeled [prerequisites](#prerequisites).
 
-1. [Create Automation Infrastructure (BASH script)]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure.sh")
+### Execute Script
 
-    The [Firewall-Automation-Infrastructure.sh]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure.sh") script "dot sources" the [Firewall-Automation-Infrastructure-Variables.sh]("./scripts/Firewall-Automation/Firewall-Automation-Infrastructure-Variables.sh").  
+Once the variables are updated, the setup script must be run from Visual Studio Code (thick client) using Codespaces. The script does not require input parameters because the required parameters are stored as environment variables when it runs the variable script. Run this command from the top-level directory of this repository.
```

Comment:
Running this script opens up a browser tab for me to login. If this is expected, add a note about so the user is not surprised.

Reply:
Note added.